src/pointerAction.js

```diff
@@ -488,9 +490,11 @@ export function dndzone(node, options) {
             const draggableEl = node.children[idx];
             styleDraggable(draggableEl, dragDisabled);
             if (idx === shadowElIdx) {
-                morphDraggedElementToBeLike(draggedEl, draggableEl, currentMousePosition.x, currentMousePosition.y, () =>
-                    config.transformDraggedElement(draggedEl, draggedElData, idx)
-                );
+                if (!morphDisabled) {
```

Comment:
please set `config.morphDisabled = morphDisabled` as some point. this will make it available elsewehre in the code if needed (by retrieving from dzToConfig).

Reply:
Good point. Wilco.